src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+public class Node<K,V> {
```

Comment:
make it a Static Nested class, remove hash field, getters and setters - checklist issue